pkg/sdk/go/groups_test.go

```diff
@@ -7,157 +7,260 @@ import (
 	"fmt"
 	"net/http"
 	"net/http/httptest"
+	"strings"
```

Comment:
Add tests for `ListGroupUsers` and `ListGroupChannels`

Reply:
These are covered in users and channels tests respectively. Reorganization of these tests is done by: https://github.com/absmach/magistrala/pull/2195

Reply:
These are covered in users and channels tests respectively. Reorganization of these tests is done by: https://github.com/absmach/magistrala/pull/2195

Reply:
@WashingtonKK take note